.github/workflows/continous-integration.yml

```diff
@@ -233,17 +236,19 @@ jobs:
         python-version: [ 3.7, 3.8, 3.9 ]
 
     steps:
-    - name: Run DataDog Agent
-      if: needs.changes.outputs.backend == 'true' && !((matrix.python-version == 3.8 || matrix.python-version == 3.9) && matrix.os == 'windows-2019')
-      run: |
-        docker run --name dd_agent -p 8126:8126 -d -e "DD_API_KEY=${{ secrets.DD_API_KEY }}" -e "DD_INSIDE_CI=true" -e "DD_HOSTNAME=none" -e "DD_SITE=datadoghq.eu" -e GITHUB_ACTIONS=true -e CI=true datadog/agent:latest
-        docker ps --all --filter name=dd_agent --filter status=running --no-trunc --format "{{.ID}} {{.Status}}"
-        docker port dd_agent
-
     - name: Checkout git repository 🕝
       if: needs.changes.outputs.backend == 'true'
       uses: actions/checkout@v3
 
+    - name: Run DataDog Agent
```

Comment:
Does it matter that this step follows the git repo checkout step?

Reply:
Good question. As long as we start the agent before the unit test step, we are good. Plus, the datadog agent works fine. 😜